nameless/__init__.py

```diff
@@ -0,0 +1,10 @@
+import discord
+import bot
+
+class NamelessText(bot.Extension):
+    @bot.event("on_message")
+    async def replace_nameless(ctx, message):
+        """Deletes any messages containing nameless and replaces it with Nameless"""
+        if "nameless" in message.content.replace(" ", ""):
+            await message.channel.send("{}\n There, fixed that for ya".format(message.content.replace("nameless", "Nameless")))
```

Comment:
above you have
```py
"nameless" in message.content.replace(" ", "")
```
but here you don't have `.replace(" ", "")`. This can result in a situation where the if condition is true but then no change happens, for example with "hello name less" will result in "hello name less\n There, fixed that for ya"

Reply:
The `.replace(" ", "")` should just be removed